tko_web_sessions_management/main.py

```diff
@@ -206,6 +205,9 @@ def save_session(
         user = request.registry.get('res.users').browse(
             cr, request.uid, uid, request.context)
         ip = request.httprequest.headers.environ['REMOTE_ADDR']
+        if ip == '127.0.0.1' or ip == 'localhost':
+            if request.httprequest.headers.environ['HTTP_X_FORWARDED_FOR']:
+                ip = request.httprequest.headers.environ['HTTP_X_FORWARDED_FOR']
         logged_in = True
```

Comment:
Shouldn't we test instead if we have or not HTTP_X_FORWARDED_FOR header, instead test by IP "127.0.0.1" or "localhost"?